src/Parcel/Parcel.service.js

```diff
@@ -58,7 +58,7 @@ export class ParcelService {
       let data
 
       try {
-        const landData = await contract.landData(parcel.x, parcel.y)
+        const landData = await contract.getData(parcel.x, parcel.y)
```

Comment:
This is ok, but keep in mind that ultimately, this should call `landData` (like it does on master). To avoid having a breaking merge, do you want to rebase master?